WatchApp Extension/Managers/ComplicationChartManager.swift

```diff
@@ -117,15 +118,50 @@ final class ComplicationChartManager {
     }
 
     private func drawOverridesIfNeeded(in context: CGContext, using scaler: GlucoseChartScaler) {
-        guard
-            let override = data?.activeScheduleOverride,
-            let overrideHashable = TemporaryScheduleOverrideHashable(override)
-        else {
-            return
+        let overrideColor = UIColor.glucose.withAlphaComponent(0.4).cgColor
+        let extendedOverrideColor = UIColor.glucose.withAlphaComponent(0.25).cgColor
+        let spannedInterval = scaler.dates
+
+        func drawOverride(
```

Comment:
Why a `func` within a `func`? Why not just another `private func` on the class?

Reply:
Oh, is it to get access to `context` and `scaler`?

Reply:
Yes, this function uses declarations from the outer function scope.